src/power.jl

```diff
@@ -67,11 +67,12 @@ function IntervalMatrixPower(M::IntervalMatrix{T}) where {T}
     return IntervalMatrixPower(M, M, 1)
 end
 
-function IntervalMatrixPower(M::IntervalMatrix{T}, k::Int) where {T}
+function IntervalMatrixPower(M::IntervalMatrix{T}, k::Int;
+                             algorithm::String="power") where {T}
```

Comment:
Should algorithm be a field of the type?

Reply:
I was also thinking about that. It is more general to let you choose at every step, but we could at least have a fallback algorithm defined in the type maybe?

Reply:
At first sight it looks better to me if the algorithm is in the type, since it is a property of how to get to `M^k`. (of course i'm  assuming that there is no mixed algorithm, but the "mixed" algorithm can be an algorithm by itself).

Reply:
Anyway, we can revisit this later.

Reply:
> (Of course i'm assuming that there is no mixed algorithm, but the "mixed" algorithm can be an algorithm by itself).

Yes and no. If the mixed algorithm follows a strict pattern, it can be added as another algorithm. But adding a new algorithm for any pattern is not sustainable. Furthermore, you can imagine patterns that depend on user knowledge (like "next I need more precision") that you cannot express in a static algorithm. So I conclude that a `string` as a field is not sufficient. If we make the algorithm a field, it should be its own struct `<:IntervalMatrixPowerAlgorithm` that you can influence from outside.

Reply:
I opened #139.